code/datums/jobs/job/job.dm

```diff
@@ -312,3 +313,8 @@ GLOBAL_PROTECT(exp_specialmap)
 
 /datum/job/proc/handle_special_preview(client/parent)
 	return FALSE
+
+/datum/job/xenomorph/occupy_job_positions(amount, respawn)
+	if((total_positions - current_positions - amount) < 0)//Bad, really bad
+		return FALSE
+	return ..()
```

Comment:
is this hack still needed

Reply:
Probably not. I still want to keep this, it's a hard check in case i missed a weird edge case or whatever

Reply:
You shouldnt let code silently fail like this, remove it